packages/analytics/dashboard-renderer/src/composables/useDashboardInternalContext.ts

```diff
@@ -0,0 +1,101 @@
+import { computed, getCurrentInstance, ref, type Ref, type DeepReadonly } from 'vue'
```

Comment:
Nit: not a blocker - can followup may be
Should we have a unit test for this composable?